### PR TITLE
Make test_approve not rely on specific log level selection

### PR DESCRIPTION
**Pull request by okurz**
Unit tests should not be too specific and we should be quite limited
with looking into actual log messages. This commit relaxes the checks
using caplog to the point what should be actually necessary for each
individual test step and not more. With this one can select an arbitrary
log level when calling pytest for easier development.

### CODE REVIEW

**Review by kalikiana — APPROVED** [review]
Nice! That's definitely something that makes working on the code easier since several tests don't need to be updated when adding something new.

File: tests/test_approve.py [hunk 1]
```diff
@@ -193,7 +193,7 @@ def test_single_incident(fake_qem, caplog):
         ],
     )
     approver(incident=1)
-    assert len(caplog.records) == 5
+    assert len(caplog.records) >= 1, "we rely on log messages"
```
Comment (mpagot) [inline tests/test_approve.py:196]: I suggest to also add short docstring to document which behavior this test is testing

Comment (okurz) [reply tests/test_approve.py:196]: Yes but I don't think this comment helps here.

Comment (kalikiana) [reply tests/test_approve.py:196]: The following asserts check that the expected messages are seen. Arguably this check is completely redundant.

Comment (okurz) [reply tests/test_approve.py:196]: Well, the check for the length of the arrays checks helps to prevent confusing IndexError exceptions

**Review by Martchus — APPROVED** [review]

**Pull request merged**